python/api/src/araali/aws.py

```diff
@@ -21,6 +21,41 @@ def cf_rm(name):
 def cf_add_vm():
     print("adding vm")
 
+def cf_launch_fortified_vm(stack_name, email, key_pair, ami_id):
+    if ami_id is None or ami_id == "":
+        # Launch latest AmazonLinux AMI
+        template_url = "https://s3.us-west-1.amazonaws.com/araalinetworks.test/quickstart_vm/AraaliVMQuickstartStack.template.json"
+        return utils.run_command_logfailure(
+            "aws cloudformation create-stack --stack-name %s --template-url %s\
+            --parameters ParameterKey=KeyName,ParameterValue=%s ParameterKey=email,ParameterValue=%s" % (
+            stack_name, template_url, key_pair, email)
+        )
+    else:
+        # Use custom AMI ID (For internal usage)
```

Comment:
do you really need to use two different cf's? Why cant you have a default value for ami_id and have one Cf?